docs/Actions.md

```diff
@@ -1492,6 +1493,24 @@ before_all do
 end
 ```
 
+## Cordova
+
+### cordova_get_config_value
+
+Get a value from a config file, which can be used to fetch the app identifier
+and more information about your app
+
+```ruby
+identifier = cordova_get_config_value(key: 'id')
+puts identifier # => tools.fastlane.cordova
+
+# path defaults to
+name = cordova_get_config_value(
+  path: 'cordova/config.xml',
+  key: 'name').first
+puts name.first # => Cordova App
```

Comment:
ok, i'm using xmlsimple to parse the xml, so it return a simple ruby object that we can use in the `Fastfile`

The thing is that node inside a element are returned as an array.
In this case

```xml
<?xml version='1.0' encoding='utf-8'?>
<widget id="tools.fastlane.cordova" version="1.1.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
    <name>Cordova App</name>
    <description />
....
</widget>
```

The action is returning the a specific key (`key` parameter in the action). An in this example the real value is of `name` is an array, thats why I was using `.first`

```ruby
cordova_get_config_value(key: 'id') #  ["Cordova App"]
```